### DISCUSSION TITLE
What is the maximum height the rangefinder is used up to

### DISCUSSION THREAD

**Discussion by and-sh** [iNavFlight/inav#10463] [Q&A]
Usually cheap rangefinder gives wrong value above some height. How to limit height at which rangefinder is used?

**Comment by anti-vaxxer** [comment]
to limit it to 2 meters for example enter this in CLI
set nav_max_terrain_follow_alt = 200
set inav_max_surface_altitude = 200

**Comment by and-sh** [comment]
Will rangefinder be used below this altitude for althold when surface mode is off?

**Reply by sensei-hacker** [reply]
> Will rangefinder be used below this altitude for althold when surface mode is off?

No. That would cause it to fall into a hole / ravine and you would lose radio signal.

**Reply by and-sh** [reply]
Thank you
